### PR TITLE
Metavix/adding server changer

### PR DESCRIPTION
**Pull request by Metavix**
@woakas  @AgustinPelaez pongo el nuevo readme con algunos cambios de nombres.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,25 +147,33 @@ void loop() {
...
+```c
 ```cpp
 // This example is to save values with a setted data source name
 
 #include "Ubidots/Ubidots.h"
 
 
 #define TOKEN "Your_Token_Here"  // Put here your Ubidots TOKEN
-#define DATA_SOURCE_NAME "Your_Data_Source_Name"
+#define DATA_SOURCE_Tag "Your_Data_Source_Tag"
```
Comment (woakas) [inline README.md]: The define must be **DATA_SOURCE_TAG**

File: firmware/Ubidots.cpp [hunk 2]
```diff
@@ -60,7 +61,7 @@ void Ubidots::setMethod(uint8_t method) {
  * @return true uppon succes
  */
 
-bool Ubidots::setDatasourceName(char* dsName) {
+bool Ubidots::setDataSourceName(char* dsName) {
```
Comment (woakas) [inline firmware/Ubidots.cpp]: @Metavix Why change this name?

Comment (Metavix) [inline firmware/Ubidots.cpp]: Because Data Source in Ubidots are two words and it was not one word

File: README.md (hunk 1 above)
Comment (Metavix) [inline README.md]: Fixed

**Comment by Metavix** [comment]
@woakas wath me! please

File: firmware/Ubidots.cpp (hunk 2 above)
Comment (Metavix) [inline firmware/Ubidots.cpp]: @woakas  fixed

**Pull request merged**